Backend/GethAPI/index.js

```diff
@@ -0,0 +1,61 @@
+const {
+    getBlockData,
+    getBlockNumber,
+    getBlockTransactionsCount,
+    getTransaction
+} = require("./gethapi.js")
+const express = require('express');
+const app = express();
+const PORT = 8080;
+
+app.use(express.json());
+app.listen(PORT);
+
+
+app.get('/getBlockNumber', async(req, res) => {
+    let blockNumber = await getBlockNumber();
+    if (blockNumber) {
+        res.status(201).send({
+            text: `Block Number: ${blockNumber}`
+        });
+    } else {
+        res.sendStatus(404);
+    }
+
+});
+
+app.post('/getBlockTransactionsCount', async(req, res) => {
+    const { hash } = req.body;
+    let blockTransactionsCount = await getBlockTransactionsCount(hash);
+    if (blockTransactionsCount) {
+        res.status(201).send({
+            text: `Transactions count: ${blockTransactionsCount}`
+        });
+    } else {
+        res.sendStatus(404);
+    }
+});
+
+app.post('/getTransaction', async(req, res) => {
+    const { hash } = req.body;
+    let transaction = await getTransaction(hash);
+    if (transaction) {
+        res.status(201).send({
+            text: `Transaction: ${transaction}`
+        });
+    } else {
+        res.sendStatus(404);
+    }
+});
+
+app.post('/getBlockData', async(req, res) => {
+    const { number } = req.body;
+    let blockData = await getBlockData(number);
+    if (blockData) {
+        res.status(201).send({
+            text: `Block Data: ${JSON.stringify(blockData)}`
+        });
+    } else {
+        res.sendStatus(404);
```

Comment:
The endpoints work fine in Postman. The only problem is that we don't have enough blockchain information, because running a full node on the Ethereum blockchain will take as around one week and approximately 200 GB of data.